hosts_example.yml

```diff
@@ -212,6 +212,43 @@ all:
     # mds_super_user_password: <ldap password>
     ##
     ##
+
+    #### LDAP Configuration
+    ##
+    ## You will need to specify the following authentication and lookup details for your LDAP server in order for RBAC to lookup and validate users.
+    ##
+    # rbac_ldap_url: <URL for LDAP server>
+    # rbac_ldap_port: <Port number or LDAP server, typically 389>
+    # rbac_ldap_principal: <Principal for authenticating to LDAP server>
+    # rbac_ldap_principal_password: <Password for LDAP server principal>
+    # rbac_ldap_user_search_base: <User search base dn, eg. CN=Users,DC=qe,DC=confluent,DC=io>
+    # rbac_ldap_group_search_base: <Group search base dn, eg. CN=Users,DC=qe,DC=confluent,DC=io>
+    ##
+    ## If you are using LDAPS, please uncomment the following:
+    ##
+    # rbac_ldaps: true
+    #
+    ## Note: We assume that the certificates for your Kafka cluster have been signed by the same CA as those for your LDAP server.
+    ##
+    ## LDAP RBAC users for Component Authentication:
```

Comment:
Maybe call this "Mandatory LDAP Users" and include the mds user here to keep them all grouped